File: benchmarks/fixtures/polyglot-services/src/UserService.swift
import Foundation

class UserService {
    func getUser(id: String) -> User? {
        return users[id]
    }
    
    func createUser(name: String, email: String) -> User {
        let user = User(id: UUID().uuidString, name: name, email: email)
        users[user.id] = user
        return user
    }
    
    func deleteUser(id: String) -> Bool {
        return users.removeValue(forKey: id) != nil
    }
}

struct User {
    let id: String
    let name: String
    let email: String
}

File: benchmarks/fixtures/polyglot-services/src/user_service.dart
class UserService {
    String name;
    
    UserService(this.name);
    
    String getUser() => 'User: $name';
    
    int calculate(int x) => x * 2;
}

void main() {
    var service = UserService('test');
    print(service.getUser());
}